tests/backup/backup_rbac_test.go

```diff
@@ -1521,12 +2005,27 @@ var _ = Describe("{VerifyRBACForAppUser}", func() {
 			err := SetSourceKubeConfig()
 			log.FailOnError(err, "Unable to switch context to source cluster [%s]", SourceClusterName)
 		}()
+		backupDriver := Inst().Backup
+		bkpScheduleEnumerateReq := &api.BackupScheduleEnumerateRequest{
+			OrgId: orgID,
+		}
+		nonAdminCtx, err := backup.GetNonAdminCtx(appUser, commonPassword)
+		log.FailOnError(err, "failed to fetch user %s ctx", appUser)
+		currentSchedules, err := backupDriver.EnumerateBackupSchedule(nonAdminCtx, bkpScheduleEnumerateReq)
+		log.FailOnError(err, "Getting a list of all schedules")
```

Comment:
dash.VerifySafely?

Reply:
Sorry, had missed it. It's included now.